### PR TITLE
gets a review list page

### PR DESCRIPTION
**Pull request by theotherbali**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -28,11 +17,22 @@
...
-  }
+
+.ReviewCard{
+  display: grid;
+  grid-template-columns: 20vw auto;
+  grid-template-areas:
+  "image title";
+  border: #282c34;
+  border-width: 10vw;
+}
```
Comment (dav3rid) [inline src/App.css:28]: Can see you've added border details here but it doesn't seem to be appearing - you'll need to add a `border-style` property for it to appear like `border-style: solid` - if you do this however, you may notice that `10vw` is probably too much! Consider lowering that value or using a different unit

File: src/Components/ReviewCard.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const ReviewCard = ({ title, designer, owner, review_img_url, review_body, category, created_at, votes}) =>{
+    return(
+        <li className="ReviewCard" >
+            <p className="reviewTitle">{title}</p>
+            <img className="reviewImage" src={`${review_img_url}`} alt={`${title}`} />
```
Comment (dav3rid) [inline src/Components/ReviewCard.jsx:5]: Not hurting anyone, but you don't need template literals here eg `src={review_img_url}`

File: src/Components/ReviewList.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    if(isLoading) {
+        return <p>Loading...</p>
+    }
+
+    return (
+        <main key="ReviewListingPage">
+            <h2>a list of reviews</h2>
+            <ul className="ReviewList" key="ReviewList" >
+                {reviews.map((review) => (<ReviewCard key={review.id} title={review.title} designer={review.designer} owner={review.designer} review_img_url={review.review_img_url} review_body={review.review_body} category={review.category} created_at={review.created_at} votes={review.votes}  />
+                    ))}
```
Comment (dav3rid) [inline src/Components/ReviewList.jsx:38]: If you find yourself passing all object properties as props to a component, you can simply spread them in as a shorthand eg `<ReviewCard {...review} />`

Comment (dav3rid) [reply src/Components/ReviewList.jsx:38]: Also, you're getting an error in the console regarding a missing `key` prop for the children of the map - I can see that you are passing a `key` to the `<ReviewCard />` here, but you must then pass that to the `li` element that it immediately returns

**Review by dav3rid — APPROVED** [review]

File: src/Components/ReviewList.jsx (hunk 3 above)
Comment (theotherbali) [reply src/Components/ReviewList.jsx:38]: I changed it from {...review} to attempt to solve the missing key prop :( but wasn't that apparently! will look at this and message via slack if i can't solve it

File: src/App.css (hunk 1 above)
Comment (theotherbali) [reply src/App.css:28]: was aware of it being too much but was confused by why it wasn't showing up! Thanks!

**Pull request merged**